dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/DataAnalysisController.java

```diff
@@ -673,6 +673,8 @@ private Grid generateValidationRulesReportGridFromResults( List<ValidationResult
             {
                 validationResultView.setOrganisationUnitId( organisationUnit.getUid() );
                 validationResultView.setOrganisationUnitDisplayName( organisationUnit.getDisplayName() );
+                validationResultView.setOrganisationUnitPath( organisationUnit.getPath() );
+                validationResultView.setOrganisationUnitAncestorNames( organisationUnit.getAncestorNames() );
```

Comment:
Please do a performance check here to see if performance is affected significantly, as these two calls can be very expensive, especially for lower-level org units in large org unit hierarchies.